components/MintPage/MintV2Page.tsx

```diff
@@ -19,11 +19,14 @@ const MintV2Page = () => {
   const mintRef = useRef()
   const [isMintLoading, setIsMintLoading] = useState(false)
   const [openSuccessModal, setOpenSuccessModal] = useState(false)
+  const [openSoldOutModal, setOpenSoldOutModal] = useState(false)
+
   const { mint, totalSupply, getTotalSupply } = useCre8orMintV2()
-  const { isConnected, address } = useAccount()
-  const { cre8orNumber, getCre8orNumber } = useCre8orNumber({ address })
+  const { isConnected } = useAccount()
   const { publicSalePrice } = useMintProvider()
 
+  const isSoldout = useMemo(() => parseInt(totalSupply, 10) === 44444, [totalSupply])
```

Comment:
make a constant variable called `MAX_SUPPLY` with value of `4444` easier to understand.
I prefer not to use magic numbers.

Reply:
fixed 👍